test/irb/test_debug_cmd.rb

```diff
@@ -16,13 +16,16 @@ class DebugCommandTestCase < TestCase
     }
 
     def setup
-      if ENV["GITHUB_ACTION_REPOSITORY"] != "ruby/irb"
```

Comment:
Using this env means it's disabled locally as well. But AFIAK, adding `ENV["CI"]` to the condition may not work because not all Ruby CI environments have it set.